src/runtime/hexagon/sim/driver/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is a standalone `CMakeLists.txt`.  The files in contrib seem to be some cmake "sub-files" for including in other cmake files.  Could you elaborate on what you were suggesting?

Reply:
[VTA.cmake](https://github.com/apache/incubator-tvm/blob/master/cmake/modules/VTA.cmake) is an example of CMake config for building standalone lib. Also, given that it's unfriendly to require users to navigate to src/runtime/hexagon/sim/driver, and having CMakeLists.txt exist under src directory seems a bit odd to me, I think it's better to put this under `cmake`.

Reply:
1. It's not just for a standalone lib/program, it's a standalone CMakeLists.txt.  All other cmake files in the cmake directory are included in the main CMakeLists.txt.  This one isn't.
2. The driver has to be compiled with hexagon-clang.  That doesn't correlate with any other compilation that uses hexagon-clang (libtvm_runtime.so needs to be compiled for Hexagon in both cases, when using a simulator and a hardware device), so it shouldn't really be included in the main CMakeFile.txt.

I was planning to add it as a dependency to the main CMakeLists.txt after it's checked in.  This would remove the need for manual running.

Reply:
I don't have strong opinion on this, but suggest to improve the experience of potential users. They suppose to turn USE_HEXAGON_SIM_DRIVER flag ON in the [config.cmake](https://github.com/apache/incubator-tvm/blob/master/cmake/config.cmake) to have this driver build automatically, instead of navigating to src/runtime/hexagon/sim/driver and start another build configure again.

To achieve this goal, 

1. Rename this file to HexagonSimDriver.cmake, and have it included in the main CMakeLists.txt
2. The following CMake config would be helpful to locate the hexagon-clang and have the proposed sim driver correctly compiled.

https://github.com/apache/incubator-tvm/blob/19f322d70335c13c0d4f7af15c684bd414b90b40/cmake/modules/Hexagon.cmake#L22

Reply:
I've already added a commit that makes this a dependency and now it's compiled automatically.  Doesn't that address your concerns about user-friendliness?

Reply:
sure, thanks.